### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sarvottam7kr**
Cleaning command needs superuser permission and for downloading FreeBSD docs, using `local` doesn't work.  The command should be the same for all docs by using `source-<wikiname>`

### CODE REVIEW

**Comment by filiparag** [comment]
Hello Sarru,

I fixed `local` target in commit [`dc471ee`](https://github.com/filiparag/wikiman/commit/dc471ee31df743d4be23b34d77482ea1cb6c461e#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52). Please report if the error persists.
As of why that rule even even exists – it's because *wikiman* uses `/usr/local` instead of `/usr` installation path on FreeBSD. If it defaulted to `/usr/local` for `source-fbsd`, it would break installing FreeBSD documentation on Linux. If you want to ignore `local`, you can, but it has to be consistent with omission of `local` rule in program installation step.
If you have an idea how to make installation prefix autodetection in `Makefile` without adding much complexity, feel invited to do so.

Thanks for fixing documentation for `clean` rule. I totally forgot to add `sudo`.

File: README.md [hunk 1]
```diff
@@ -116,12 +116,12 @@ curl -L 'https://raw.githubusercontent.com/filiparag/wikiman/master/Makefile' -o
 # Example for Linux: install Arch Wiki
 make -f ./wikiman-makefile source-arch
 sudo make -f ./wikiman-makefile source-install
-make -f ./wikiman-makefile clean
+sudo make -f ./wikiman-makefile clean
 
 # Example for BSD: install FreeBSD Documentation
-make -f ./wikiman-makefile source-fbsd local
+make -f ./wikiman-makefile source-fbsd
```
Comment (filiparag) [inline README.md]: Please add back `local` to this line, so I can merge the edit.

**Review by filiparag — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sarvottam7kr) [reply README.md]: done

**Review by filiparag — APPROVED** [review]

**Comment by filiparag** [comment]
Thanks for your contribution!

**Pull request merged**